File: Selenium/Activities/Activity_20.java
package Activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_20 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("The Page title is: " + driver.getTitle());
        driver.findElement(By.id("prompt")).click();
        Alert alert = driver.switchTo().alert();
        System.out.println("The alert message is: " + alert.getText());
        alert.sendKeys("Awesome");
        alert.accept();
        driver.quit();
    }
}

File: Selenium/Activities/Activity_20.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By


with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("The page title is: ", driver.title)
    driver.maximize_window()
    driver.find_element(By.ID, "prompt").click()
    alert = driver.switch_to.alert
    print(alert.text)
    alert.send_keys("awesome")
    time.sleep(5)
    alert.accept()

    driver.quit()